apps/tidesandtemp/tides_and_temp.star

```diff
@@ -0,0 +1,202 @@
+
+    return render.Root(
+        delay = 350,
+        child = render.Row(
+            [
+                render.Column(
+                    children = disp_predictions,
+                    main_align = "center",
+                    cross_align = "center",
+                    expanded = True,
+                ),
+                render.Column(
+                    children = [
+                        render.Image(src = img, width = 20, height = 10),
+                        # Variable box height based on current tide percentage
+                        render.Box(height = 1 + int(16 * curr_tide_pct / 100), color = "#99ccff"),
+                        render.Row(
+                            children = [
+                                render.Text(str(water_temp) + "F", color = waterTempColor, font = "tom-thumb"),
+                                render.Box(height = 1, width = 1),
+                            ],
+                        ),
+                    ],
+                    main_align = "end",
+                    cross_align = "end",
+                    expanded = True,
+                ),
+            ],
+        ),
+    )
+
+def get_schema():
+    timezones = [
+        schema.Option(display = "Hawaii (-10)", value = "Pacific/Honolulu"),
+        schema.Option(display = "Alaska (-9)", value = "America/Anchorage"),
+        schema.Option(display = "Pacific (-8)", value = "America/Los_Angeles"),
+        schema.Option(display = "Mountain (-7)", value = "America/Denver"),
+        schema.Option(display = "Central (-6)", value = "America/Chicago"),
+        schema.Option(display = "Eastern (-5)", value = "America/New_York"),
+        schema.Option(display = "Atlantic (-4)", value = "America/Halifax"),
+        schema.Option(display = "Newfoundland (-3.5)", value = "America/St_Johns"),
+        schema.Option(display = "Brazil (-3)", value = "America/Sao_Paulo"),
+        schema.Option(display = "UTC (0)", value = "UTC"),
+        schema.Option(display = "Central Europe (+1)", value = "Europe/Berlin"),
+        schema.Option(display = "Eastern Europe (+2)", value = "Europe/Moscow"),
+        schema.Option(display = "India (+5.5)", value = "Asia/Kolkata"),
+        schema.Option(display = "China (+8)", value = "Asia/Shanghai"),
+        schema.Option(display = "Japan (+9)", value = "Asia/Tokyo"),
+        schema.Option(display = "Australia Eastern (+10)", value = "Australia/Sydney"),
+        schema.Option(display = "New Zealand (+12)", value = "Pacific/Auckland"),
+    ]
+
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Dropdown(
```

Comment:
A common pattern for getting timezone data is to use `schema.Location`. That allows the user to select a specific location, and gives you the associated timezone in the location object.

Reply:
Also, you automatically get the device's configured location in the magic '$tz' config variable. Not sure if that's good enough or if it's important for this to be fully configurable.

Reply:
I thought about using the `schema.Location` option or the magic `$tz` config variable, but I thought it may be advantageous for the user to have more direct control over setting the timezone.

Reply:
Got it. In that case I still think `Location` is a good option to consider. Gives you handling of e.g. daylight's savings for free, and is likely more intuitive for the user (most people don't think much about their timezone's current offset from UTC for some reason) =). But it's your call!

Reply:
I think you're right, I will switch to using Location

Reply:
Switched to using `schema.Location` now